{
    "type": "bundle",
    "id": "bundle--917e20ff-af76-4630-8538-4c14e8259993",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--61ab3138-583b-4255-8510-8c4ec765df62",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-12-30T01:13:11.880187Z",
            "modified": "2024-12-30T01:13:11.880187Z",
            "name": "CVE-2024-56606",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\naf_packet: avoid erroring out after sock_init_data() in packet_create()\n\nAfter sock_init_data() the allocated sk object is attached to the provided\nsock object. On error, packet_create() frees the sk object leaving the\ndangling pointer in the sock object on return. Some other code may try\nto use this pointer and cause use-after-free.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-56606"
                }
            ]
        }
    ]
}